Content.Server/Paper/PaperSystem.cs

```diff
@@ -154,6 +155,16 @@ private void OnInputTextMessage(EntityUid uid, PaperComponent paperComp, PaperIn
                 _adminLogger.Add(LogType.Chat, LogImpact.Low,
                     $"{ToPrettyString(args.Actor):player} has written on {ToPrettyString(uid):entity} the following text: {args.Text}");
 
+                //If the user is a mime make their writings incomprehensible 
+                if(HasComp<MimePowersComponent>(args.Actor))
+                {
+                  paperComp.Content = new string(
+                      args.Text
+                      .OrderBy(c => Guid.NewGuid())
+                      .ToArray()
+                      );
+                }
+
                 _audio.PlayPvs(paperComp.Sound, uid);
```

Comment:
Tightly coupling this to paper is an awful idea.

Reply:
Is there a paper event I can use instead?